driver-adapters/tidb-node-basic/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -eu
+
+pnpm install
+pnpm prisma generate
```

Comment:
Should we invoke `prisma db push --force-reset` here since it won't be handled by CI automatically?

Reply:
Since the workflow is already passing the test, it means the db is already in sync

Reply:
I think this is just copy-pasted from the other tests - and seems to rely on the database to have been set up manually indeed.

I am ok with that.